app.py

```diff
@@ -414,6 +415,9 @@ def setup_language():
     # front-end is fixed based on this value
     g.dir = static_babel_content.TEXT_DIRECTIONS.get(g.lang, 'ltr')
 
+    # True if it is a Latin alphabet, False if not
```

Comment:
Love this comment! This is exactly the type of thing that is very easy to understand/explain and superhard to get from the code!